opm/material/fluidsystems/ThreeComponentFluidSystem.hh

```diff
@@ -194,8 +194,7 @@ namespace Opm {
                                  unsigned phaseIdx)
         {
             // Use LBC method to calculate viscosity
-            LhsEval mu;
-            mu = ViscosityModel::LBC(fluidState, paramCache, phaseIdx); 
+            const auto mu = decay<LhsEval>(ViscosityModel::LBC(fluidState, paramCache, phaseIdx));
             return mu;
```

Comment:
If you're changing this anyway, why not just write it as
```C++
return decay<LhsEval>(...);
```
instead?

Reply:
We can do that. It is just a long return sentence. I thought it might be easier to read to break it down.